internal/clusterconfig/clusterconfig.go

```diff
@@ -162,9 +153,73 @@ func (r *UpgradeClusterConfigGather) GetKubeadminPasswordHash(ctx context.Contex
 	return kubeadminPasswordHash, nil
 }
 
+type AdditionalTrustBundle struct {
```

Comment:
why not to use seedreconfig.AdditionalTrustBundle ? Seems totally the same or i missed something?